docs/specification.md

```diff
@@ -184,14 +253,16 @@ exist.
   - `foo.default.svc.cluster.local. 10 IN CNAME www.example.com.`
   - `www.example.com. 28715 IN A 192.0.2.53`
 
-### 2.6 - Deprecated Records
-Kube-DNS versions prior to implementation of this specification also replied
-with an `A` record of the form below for any values of `<a>`, `<b>`, `<c>`, and `<d>` between 0 and 255:
```

Comment:
I don't know if it's ok to remove the old records, but seems reasonable because the deprecation was announced in Dec 2016 based on git blame.
My rationale is that because they are IPv4 only it may be problematic having different records for each IP family.

Reply:
We should send something out on the sig-network mailing list as FYI
Did the various implementations remove this record yet?

Reply:
No, they all still support it. It's a config option in CoreDNS, I don't think there is any way to turn it off in kube-dns. I don't know about, say, OpenShift which I believe has its own implementation (at least it used to).